docs/pages/machine-id/reference/cli.mdx

```diff
@@ -0,0 +1,72 @@
+|----------------------|------------------------------------------------------------------------------------------------|
+| `-d/--debug`         | Enable verbose logging to stderr.                                                              |
+| `-c/--config`        | Path to a configuration file. Defaults to `/etc/tbot.yaml` if unspecified.                     |
+| `-a/--auth-server`   | Address of the Teleport Auth Server (On-Prem installs) or Proxy Server (Cloud installs).       |
+| `--token`            | A bot join token, if attempting to onboard a new bot; used on first connect.                   |
+| `--ca-pin`           | CA pin to validate the auth server.                                                            |
+| `--data-dir`         | Directory to store internal bot data. Access should be limited. See `tbot init` for details.   |
+| `--destination-dir`  | Directory to write short-lived machine certificates.                                           |
+| `--certificate-ttl`  | TTL of short-lived machine certificates.                                                       |
+| `--renewal-interval` | Interval at which short-lived certificates are renewed; must be less than the certificate TTL. |
+| `--join-method`      | Method to use to join the cluster, can be `token` or `iam`.                                    |
+| `--one-shot`         | If set, quit after the first renewal.                                                          |
+
+## `tbot init`
+
+Initialize a certificate destination directory for writes from a separate bot
+user. Useful when running the bot under an isolated user account.
+
+```code
+$ tbot init \
+    --destination-dir=/opt/machine-id \
+    --bot-user=ubuntu \
+    --owner=ubuntu:ubuntu \
+    --reader-user=jenkins
+```
+
+| Flag                 | Description                                                                                           |
+|----------------------|-------------------------------------------------------------------------------------------------------|
+| `-d/--debug`         | Enable verbose logging to stderr.                                                                     |
+| `-c/--config`        | Path to a configuration file. Defaults to `/etc/tbot.yaml` if unspecified.                            |
+| `--destination-dir`  | Directory to write short-lived machine certificates to.                                               |
+| `--owner`            | Defines Linux user:group owner of `--destination-dir`. Defaults to `nobody:nobody` if unspecified.    |
+| `--bot-user`         | When using ACLs, defines Linux user that can write short-lived certificates to `--destination-dir`.   |
+| `--reader-user`      | When using ACLs, defines Linux user that will read short-lived certificates from `--destination-dir`. |
+| `--init-dir`         | If using a config file and multiple destinations are configured, specify which to initialize.         |
```

Comment:
Tim: I copied this over from the CLI flag help, but to be honest, I didn't quite understand the purpose of this flag.

Reply:
Yeah, it's a bit vague, especially since we also have `--destination-dir`. The two flags control how we handle configuration:
 - `--destination-dir` adds an otherwise-unconfigured destination dir just like `tbot start` when there's no config file, potentially overriding destinations if they exist in your config file.
 - `--init-dir` just selects which directory to initialize from your config file without potentially altering your config. This is optional if using a config file with only 1 destination configured, since it will infer it automatically.

Admittedly having this split across two flags is a bit of a kludge. I'd like to try merging them together at some point but there were some hiccups integrating that with our config parsing.